src/products/ProductMapper.js

```diff
@@ -126,7 +126,7 @@ class ProductMapper {
         } else {
             variants = [
                 new ProductVariant.Builder()
-                    .withAvailable(true) // TODO: Get actual value from backend
+                    .withAvailable(product.stock_status == "IN_STOCK")
```

Comment:
Why use `==` here and `===` a few lines below? Please use only strong comparison operators.